packages/components/src/accordion/test/index.js

```diff
@@ -50,4 +50,22 @@ describe( 'Accordion', () => {
 		expect( screen.queryByText( '10000' ) ).toBeInTheDocument();
 		expect( screen.queryByText( '20000' ) ).toBeInTheDocument();
 	} );
+
```

Comment:
Looking over the tests you've added they look great. Good stuff!

Reply:
> There are a few other things, I'm also considering maybe it just wasn't clear that there was some redesign work to do here, so sorry about any ambiguity there. You can check out the designs (figma links at bottom) [here](pbIJXs-ma-p2-p2).

Thanks I did look at this before and I probably should off asked how to get the specific pixel amounts and such. I just noticed I never saw them as I didn't create an account for Figma.

Reply:
@louwie17 did you get sorted out with Figma access? There are a few tools in there for measuring distance between objects in the designs, let me know if you need help with any of that.

Reply:
@samueljseay I did figure that out, although a little overview of the tools and how to's would certainly be helpful. We can do that during our 1:1 to tomorrow?